guides/assets/stylesrc/_main.scss

```diff
@@ -135,7 +146,7 @@ body.guide {
   } // h1
 
   h2 {
-    font-family: 'Calibre', sans-serif;
+    font-family: 'Calibre', "Noto Sans Arabic", "Heebo", sans-serif;
```

Comment:
Here are the other fonts I mentioned, I think these would be covered by the declarations at the bottom.

Reply:
Cleaned up in 91ddfcc0